Packages/com.unity.inputsystem/InputSystem/Editor/AssetEditor/InputActionTreeView.cs

```diff
@@ -1232,6 +1232,7 @@ private bool ReloadIfSerializedObjectHasBeenChanged()
         public bool drawMinusButton { get; set; }
         public float foldoutOffset { get; set; }
 
+        public InputActionTreeView viewToAddActions { get; set; }
```

Comment:
Would prefer for `InputActionTreeView` to have an `onHandleAddNewAction` delegate property or something like that.

```
        public void AddNewAction(SerializedProperty actionMapProperty)
        {
            if (onHandleAddNewAction != null)
                onHandleAddNewAction(actionMapProperty);
            else
            {
                //...
            }
        }
```

Reply:
Agreed, that's much better. Changed.